modules/executive/components/SpellEffectsTab.tsx

```diff
@@ -67,13 +83,13 @@ export function SpellEffectsTab({
         </Text>
 
         <Box
-          sx={{
-            background: '#F6F8F9',
+          sx={theme => ({
+            background: theme.colors?.background,
```

Comment:
This fixes a bug with dark mode. I forgot how much`@reach` makes it a pain to use our theme, but this is how it can be done. Supposedly it is compatible with emotion so I have to look into how we can integrate our theme file with reach.